Mage.Sets/src/mage/cards/s/SacellumGodspeaker.java

```diff
@@ -90,6 +91,7 @@ public Mana produceMana(Game game, Ability source) {
         if (game != null) {
             TargetCardInHand target = new TargetCardInHand(0, Integer.MAX_VALUE, filter);
             if (target.choose(Outcome.Benefit, source.getControllerId(), source.getSourceId(), source, game)) {
+                game.getPlayer(source.getControllerId()).revealCards(source, new CardsImpl(target.getTargets()), game);
```

Comment:
Each player and other objects from UUID must be checked on null. Controller can be checked in the start of the method by standard code:
``` java
Player controller = game.getPlayer(source.getControllerId());
if (controller == null) {
    return false;
}
```

Reply:
Controller is checked for null inside target.choose, should I check it manually anyway?

Reply:
Yes, you must check it at the start. It's a standard code style. Also must replace:
`game.getPlayer(source.getControllerId()).revealCards`
to
`controller.revealCards`

Reply:
Used `if (controller != null) {` style to match with the surrounding code, if you prefer the early return style let me know (and if so, should I change other uses to that as well)?

Reply:
I definitely prefer the flattened style with an early return. I think it's easier to follow. There's a lot of existing code that uses highly nested if-statements, and there's no requirement to change that, but if you're writing/rewriting new logic, I recommend the flattened style.